content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -589,264 +593,265 @@ you can also create debit Notes. For this just use the button “Add Debit Note
+contacts, and products. In the general configurations, it is now necessary to add new credentials
+that are acquired in the SUNAT portal.
 
 Flow
 ****
 
 .. image:: peru/peru_flow_remision.png
-   :align: center 
+   :align: center
    :alt: Flow for Guia de Remision emission
-   
+
 Cancellations
-^^^^^^^^^^^^^
+*************
 
-Currently, the SUNAT no longer uses the term *“Anula”*, but now uses the term *“Dar de baja”*. 
-Therefore, **both** the sender and the carrier can cancel the electronic waybill, *so long as 
-the following conditions are met*: 
+Currently, the SUNAT no longer uses the term "Anula", but now uses the term "Dar de baja" for
+cancellations. Therefore, **both** the sender and the carrier can cancel the electronic waybill, so
+long as the following conditions are met:
 
-- If the shipment has not yet been initiated
-- If the shipment has been initiated, but the receiver must be changed before reaching 
-the final destination.
+- The shipment has not been initiated.
+- If the shipment has been initiated, the receiver **must** be changed before reaching the final
+  destination.
 
 Testing
-^^^^^^^
-
-The SUNAT did not release any test environment; **this means that any Delivery Guides that 
-were generated by mistake will be sent to the SUNAT**.
+*******
 
-If by mistake the Waybill was created in this environment, it will be necessary to be deleted 
-from the SUNAT portal.
+The SUNAT did not release a test environment. This means that any delivery guides that were
+generated by mistake **will** be sent to the SUNAT.
 
-Odoo Configuration
-******************
+If by mistake the waybill was created in this environment, it is necessary to delete it from the
+SUNAT portal.
 
-.. important::
-   - Electronic Sender’s Waybill is currently the only supported type of Waybill in Odoo.
-   - The delivery guide is dependant on the **Inventory App**, the `l10n_pe_edi` and 
-`l10n_pe` modules.
-   
-After following the steps to configure the :doc:`Electronic invoicing <accounting_settings>` 
-and the :doc:`Master data <configura_master_data>`, go to **Apps**, **clear** any default 
-filters set, search for the `l10n_pe_edi_stock_20` module and click the *Activate* button.
-
-Next, you will need to *obtain* the Client ID and Client Secret. To do this, follow the 
-instructions in this `link <https://cpe.sunat.gob.pe/sites/default/files/inline-files/Manual_Servicios_GRE.pdf>`_.
+Configuration
+~~~~~~~~~~~~~
 
 .. important::
-   Keep in mind, you must generate a second user for electronic document creation of 
-electronic documents.
-  
-To obtain the Client ID / Secret, enter the **SUNAT Portal**, and look in the “SOL” 
-menu for the **SUNAT API Credentials Management** section:
+   - Electronic sender's waybill is currently the only supported type of waybill in Odoo.
+   - The delivery guide is dependant on the Odoo *Inventory* app, the :guilabel:`l10n_pe_edi` and
```

Comment:
![image](https://github.com/odoo/documentation/assets/89666460/6889881c-0aca-4574-8651-5564f0fb49da)

"and l10n_pe modules" should be inside the important block